x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/metric_distribution_chart/metric_distribution_chart.tsx

```diff
@@ -82,7 +82,10 @@ export const MetricDistributionChart: FC<Props> = ({
   };
 
   return (
-    <div data-test-subj="dataVisualizerFieldDataMetricDistributionChart">
+    <div
+      data-test-subj="dataVisualizerFieldDataMetricDistributionChart"
+      className="dataGridChart__histogram"
+    >
       <Chart size={{ width, height }}>
         <Settings theme={theme} tooltip={{ headerFormatter }} />
```

Comment:
Probably not this specific line of code - but the line wrap for the tooltip on the distribution chart in the row looks off:

![image](https://user-images.githubusercontent.com/7405507/132859859-0828c0c4-55f4-4e4c-9b39-1f34a0cf0a53.png)

Reply:
Fixed here https://github.com/elastic/kibana/pull/107184/commits/d0d2289bcfa60e89ad6bd3e7a784e595a6c51842